distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -284,6 +284,15 @@
                 <exclude>**/derby*</exclude>
             </excludes>
         </fileSet>
+
+        <!--Empty directory for default ingest location-->
+        <fileSet>
+            <directory>.</directory>
+            <outputDirectory>ingest</outputDirectory>
+            <excludes>
+                <exclude>*/**</exclude>
+            </excludes>
+        </fileSet>
```

Comment:
Create empty `ingest` directory in the distro

Reply:
❓ Should we set permissions for the directory properly to allow read/write?